src/MF_Modules/MFAnalog.cpp

```diff
@@ -17,16 +17,35 @@ MFAnalog::MFAnalog(uint8_t pin, analogEvent callback, const char * name, uint8_t
 }
 
 void MFAnalog::update()
-{    
+{
+/* *****************************************************************************
+*   There is a pull request to transfer the millis() to the mobiflight.cpp
+*   this will save some memory and would be also valid for tick();
+********************************************************************************/
     uint32_t now = millis();
+    if (now-_lastTick > 10) {
```

Comment:
When I saw `tick` I thought this was related to clock stuff. How about calling it something like `readPin()` or `readValue()`, and have `_lastTick` be `_lastReadTime`? Just makes it clearer what the function and variable do.

Reply:
I like `readBuffer()`, good suggestion!

Reply:
It's changed